### PR TITLE
Improving the README

### PR DESCRIPTION
**Pull request by urfolomeus**
Deploy instructions were a little vague.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,47 +29,75 @@ mailcatcher --smtp-port 1035  # needs to be running in order to catch mail
...
-  BASIC_AUTH_NAME=<username>
-  BASIC_AUTH_PASSWORD=<password>
-  HOST=<hostname>
-  GA_TRACKING_ID=<tracking_id>
-```
+  1. Create a file called `config/deploy/<env>.rb` on your local machine, and populate it similar to the `config/deploy/staging.rb` file (see the [super secret instructions](https://moneyadviceserviceuk.atlassian.net/wiki/display/DEV/Webfeeds+project-specific+instructions) if you don't yet have a `config/deploy/staging.rb` file).
+  2. Run `cap <env> setup`.
+  3. SSH into the server and navigate to `/var/www/webfeeds` and ensure that environment key on line 1 of both `db/database.yml` and `shared/config/database.yml` is `production:` and not `<env>:`.
+
+You can set up production (**BUT ONLY IF IT'S NOT ALREADY SET UP**) using just `cap production deploy`.
```
Comment (rnalexander) [inline README.md]: This is the same command as on line 103, is that right?

Comment (urfolomeus) [inline README.md]: Oops, good catch. Fixing ... :)

**Comment by urfolomeus** [comment]
@rnalexander fixed

**Comment by rnalexander** [comment]
:+1:

**Comment by rnalexander** [comment]
Much clearer, thanks!

**Comment by urfolomeus** [comment]
Excellent! I'll get it merged in and then follow the same format for Common Initial Assessment.

**Comment by Tr4pSt3R** [comment]
@urfolomeus don't forget to delete this branch?

**Comment by urfolomeus** [comment]
Thanks @Tr4pSt3R :)

**Pull request merged**